facebook-instant-articles.php

```diff
@@ -350,8 +350,7 @@ function inject_ia_markup_meta_tag() {
 		// Transform the post to an Instant Article.
 		$adapter = new Instant_Articles_Post( $post );
 		if ( $adapter->should_submit_post() ) {
-			$url = $adapter->get_canonical_url();
-			$url = add_query_arg( 'ia_markup', '1', $url );
+			$url = $adapter->get_canonical_url() . 'ia/';
```

Comment:
Hey @paulschreiber, this concatenation assumes the canonical URL ends with a "/", but that's not always the case since you can change a setting in WP to make permalinks based on query string params (or simply omit the last "/"). It'd be good to decompose the URL and make sure to add the endpoint to the right place.

Reply:
Other than this, thanks for the improvement!

Reply:
Do you suggest decomposing the URL using `parse_url()`?

Reply:
@diegoquinteiro Can you clarify?